client/my-sites/themes/themes-selection.jsx

```diff
@@ -21,13 +21,17 @@ import {
 	isThemeActive,
 	isThemePurchased
 } from 'state/themes/selectors';
+import config from 'config';
 import { FEATURE_UNLIMITED_PREMIUM_THEMES } from 'lib/plans/constants';
 import { PAGINATION_QUERY_KEYS } from 'lib/query-manager/paginated/constants';
 
 const ThemesSelection = React.createClass( {
 	propTypes: {
 		query: PropTypes.object.isRequired,
-		siteId: PropTypes.number,
+		siteId: PropTypes.oneOfType( [
+			PropTypes.number,
+			PropTypes.oneOf( [ false ] )
```

Comment:
(Or remove the prop altogether there, though we should maybe add a comment then that we're doing that to get the list of WordPress.com themes.)